website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -239,17 +305,43 @@ be one__ `primary_ip` defined for edge gateway.
 allocated
 
 <a id="edgegateway-subnet-ip-allocation"></a>
+
 ## Edge Gateway Subnet IP Allocation
 
 * `start_address` - (Required) - Start IP address of a range
 * `end_address` - (Required) - End IP address of a range
 
+<a id="edgegateway-auto-subnet"></a>
+
+## Edge Gateway Automatic IP allocation (from *any* of the defined `auto_subnet` entries)
+
+* `gateway` - (Required) - Gateway for a subnet in external network
+* `prefix_length` - (Required) - Prefix length of a subnet in external network (e.g. 24 for netmask of 255.255.255.0)
+* `primary_ip` (Required) - Is required, but only in one of defined `auto_subnet` block
```

Comment:
Do NSX-T docs provide a description about primary IP? Could we add a sentence here?

Reply:
Primary IP is not a new thing https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/nsxt_edgegateway#primary_ip

It is the IP that Edge Gateway auto-assigns (unless explicitly declared which one of the allocated pools). An Edge Gateway must get at least 1 IP address during creation

Reply:
I have rephrased it now for both `subnet_with_total_ip_count` and `allocated_ip_count` sections - check if it got any better

Reply:
It's perfect.